UInnovateApp/src/App.css

```diff
@@ -40,3 +40,6 @@
 .read-the-docs {
   color: #888;
 }
+.sidebar{
```

Comment:
This is empty, what do we need it for?

Reply:
its been removed thank you for the catch.